frontend/src/components/BCDataGrid/BCDataGridClient.jsx

```diff
@@ -17,6 +17,9 @@ import DataGridLoading from '@/components/DataGridLoading'
 // Register the required feature modules with the Grid
 ModuleRegistry.registerModules([ClientSideRowModelModule])
 
+/**
```

Comment:
Do we want to remove these deprecated components?

Reply:
yes, but there are components that are still using them.